datafusion/sql/src/unparser/expr.rs

```diff
@@ -366,6 +413,27 @@ impl Unparser<'_> {
         }
     }
 
+    pub fn expr_to_unparsed(&self, expr: &Expr) -> Result<Unparsed> {
```

Comment:
Can we please add a doc string here explaining the difference here to `expr_to_sql`?